### PR TITLE
Add security and vulnerability reporting page

### PR DESCRIPTION
**Pull request by derrickmehaffy**
#### Description of what you did:

Added Security page for reporting vulnerabilities directly to Strapi via email or slack.

It's likely the email address will need to be changed as it probably doesn't exist.

#### My PR is a:

- [ ] 💥 Breaking change
- [ ] 🐛 Bug fix
- [x] 💅 Enhancement
- [ ] 🚀 New feature

#### Main update on the:

- [ ] Admin
- [x] Documentation
- [ ] Framework
- [ ] Plugin

#### Manual testing done on the following databases:

- [x] Not applicable
- [ ] MongoDB
- [ ] MySQL
- [ ] Postgres
- [ ] SQLite

### CODE REVIEW

**Comment by lauriejim** [comment]
Hello @derrickmehaffy !
Can you also create an issue template with the same content?
With the name `🛡Security` please.
Thank you.

**Comment by derrickmehaffy** [comment]
@lauriejim yup will do

File: .github/ISSUE_TEMPLATE/SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+name: 🛡 Security
+about: Information on reporting security volnerabilities
```
Comment (lauriejim) [inline .github/ISSUE_TEMPLATE/SECURITY.md]: `volnerabilities` -> `vulnerabilities` ?

**Comment by derrickmehaffy** [comment]
Thanks :D fat fingered that one lol

**Pull request merged**